src/Http/GraphRequest.php

```diff
@@ -431,20 +428,18 @@ protected function getConcatenator()
     * @param string $proxyPort The port to forward
     * requests through. If null, a proxy is not used
     *
-    * @return \GuzzleHttp\Client The new client
+    * @return array
     */
-    protected function createGuzzleClient($proxyPort = null)
+    protected function getGuzzleOptions($proxyPort = null)
     { 
         $clientSettings = [
-            'base_uri' => $this->baseUrl,
```

Comment:
I think we should keep the base_uri override for custom Guzzle clients that are passed in...